### PR TITLE
- fix crash if import contains empty line

### PR DESCRIPTION
**Pull request by EricNeid**
Moin,
für den Code of Kutulu wollte ich mal Go und deinen Merger ausprobieren.
Wenn im Import einer Datei eine Leerzeile ist, crasht er aber.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -27,8 +29,13 @@ func main() {
 	imports := make(map[string]bool, 0)
 	for _, file := range files {
 		n := file.Name()
-		if strings.Contains(n, ".go") && !strings.Contains(n, "_test") {
-			f, err := os.Open(fmt.Sprintf("%s/%s", *in, n))
+		if n == "vendor" {
```
Comment (malud) [inline main.go:31]: I would prefer skipping directories at all here instead of the vendor one.

File: main.go [hunk 2]
```diff
@@ -9,15 +9,17 @@ import (
 	"strings"
 )
 
-var in = flag.String("in", "", "-in path")
-var out = flag.String("out", "", "-out path")
+var in string
```
Comment (malud) [inline main.go]: Could be written as one line :) `var in, out string`

File: main_test.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestMain__Simple(t *testing.T) {
```
Comment (malud) [inline main_test.go]: One `_` should be enough.

```
func Example() { ... }
func ExampleF() { ... }
func ExampleT() { ... }
func ExampleT_M() { ... }
```
see https://golang.org/pkg/testing/#pkg-overview

**Review by malud — CHANGES_REQUESTED** [review]
Thanks for your submission 👍 
Some small changes and we are good to go :-)

File: main_test.go (hunk 3 above)
Comment (EricNeid) [reply main_test.go]: Thanks for the hint. I was not aware.

File: main.go (hunk 1 above)
Comment (EricNeid) [reply main.go:31]: Still printing a message to the user if "vendor" was discovered?

**Comment by EricNeid** [comment]
I incorporated your requested changed. Please review them again.

**Review by malud — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (malud) [reply main.go:31]: Nope :) User would see a compiling error later on for missing packages.

**Pull request merged**